ios/browser/api/ads/ads_client_bridge.h

```diff
@@ -71,7 +71,7 @@
 - (void)recordP2AEvent:(const std::string&)name
                   type:(const ads::mojom::P2AEventType)type
                  value:(const std::string&)value;
-- (void)logTrainingInstance:
+- (void)AddCovariatesToDataStore:
```

Comment:
Should be lower camel case to match obj-c standards (`addCovariatesToDataStore`)